dotsBkgItem.py

```diff
@@ -67,17 +67,18 @@ def __init__(self, imgFile, parent, mirror=False):
     def mousePressEvent(self, e):
         if e.button() == Qt.RightButton:
             self.ungrabMouse()   
+
         if self.canvas.key == 'del':       # delete it
             self.canvas.scene.removeItem(self)
             self.canvas.disableSliders()
             self.canvas.buttons.btnBkgFiles.setEnabled(True)
+
         elif self.canvas.key == 'bak':    # send to back
             self.setZValue(self.sideCar.lastZval('bkg')-1)
+
         elif self.canvas.key == 'shift':
-            if self.flopped == False:
-                self.flopped = True
-            else:
-                self.flopped = False
+            # Toggle flopped state.
+            self.flopped = not self.flopped
```

Comment:
To toggle a `True` to a `False` state, you can use `not`... `True == not False` (and vice versa).